### PR TITLE
🐛 Destination Bigquery: fix bug in standard inserts for syncs >10K records

### PR DESCRIPTION
**Pull request by edgao**
there was a bug in an unreleased feature, which was intended to be gated behind an inactive flag. That flag didn't properly gate everything. Fix it.

### CODE REVIEW

**Comment by octavia-squidington-iii** [comment]
##  destination-bigquery test report (commit [`fc4336918d`](https://github.com/airbytehq/airbyte/pull/27856/commits/fc4336918d30503f9551927ad1d11a314a547288)) - ❌

⏲️  Total pipeline duration: 29mn13s 

| Step                                                                            | Result   |
|:--------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-bigquery/metadata.yaml     | ✅        |
| Connector version semver check                                                  | ✅        |
| Connector version increment check                                               | ✅        |
| QA checks                                                                       | ✅        |
| Build connector tar                                                             | ✅        |
| Build destination-bigquery docker image for platform linux/x86_64               | ✅        |
| Build airbyte/normalization:dev                                                 | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-bigquery:integrationTest | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/edgao_fix_bq/1688090331/fc4336918d30503f9551927ad1d11a314a547288/destination-bigquery/1.5.1/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-bigquery test report (commit [`fc4336918d`](https://github.com/airbytehq/airbyte/pull/27856/commits/fc4336918d30503f9551927ad1d11a314a547288)) - ✅

⏲️  Total pipeline duration: 25mn40s 

| Step                                                                            | Result   |
|:--------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-bigquery/metadata.yaml     | ✅        |
| Connector version semver check                                                  | ✅        |
| Connector version increment check                                               | ✅        |
| QA checks                                                                       | ✅        |
| Build connector tar                                                             | ✅        |
| Build destination-bigquery docker image for platform linux/x86_64               | ✅        |
| Build airbyte/normalization:dev                                                 | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-bigquery:integrationTest | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/edgao_fix_bq/1688092203/fc4336918d30503f9551927ad1d11a314a547288/destination-bigquery/1.5.1/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery test
```

File: airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryRecordConsumer.java [hunk 1]
```diff
@@ -205,10 +205,12 @@ public void close(final boolean hasFailed) {
   }
 
   private void doTypingAndDeduping(final StreamConfig stream) throws InterruptedException {
-    String suffix;
-    suffix = overwriteStreamsWithTmpTable.getOrDefault(stream.id(), "");
-    final String sql = sqlGenerator.updateTable(suffix, stream);
-    destinationHandler.execute(sql);
+    if (use1s1t) {
```
Comment (evantahler) [inline airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryRecordConsumer.java:208]: Yeah... that make sense!

**Review by evantahler — APPROVED** [review]
Thanks for finding this so fast!
Are there any tests that we should do to confirm there aren't any other problems lurking?

**Review by jbfbell — APPROVED** [review]

**Comment by octavia-squidington-iii** [comment]
##  destination-bigquery-denormalized test report (commit [`4d24db8061`](https://github.com/airbytehq/airbyte/pull/27856/commits/4d24db8061f84da73d1a1987853969b97680a3ba)) - ❌

⏲️  Total pipeline duration: 16mn54s 

| Step                                                                                         | Result   |
|:---------------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-bigquery-denormalized/metadata.yaml     | ✅        |
| Connector version semver check                                                               | ✅        |
| QA checks                                                                                    | ✅        |
| Build connector tar                                                                          | ✅        |
| Build destination-bigquery-denormalized docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-bigquery-denormalized:integrationTest | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/edgao_fix_bq/1688425603/4d24db8061f84da73d1a1987853969b97680a3ba/destination-bigquery-denormalized/1.5.2/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery-denormalized test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-bigquery test report (commit [`4d24db8061`](https://github.com/airbytehq/airbyte/pull/27856/commits/4d24db8061f84da73d1a1987853969b97680a3ba)) - ✅

⏲️  Total pipeline duration: 27mn25s 

| Step                                                                            | Result   |
|:--------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-bigquery/metadata.yaml     | ✅        |
| Connector version semver check                                                  | ✅        |
| Connector version increment check                                               | ✅        |
| QA checks                                                                       | ✅        |
| Build connector tar                                                             | ✅        |
| Build destination-bigquery docker image for platform linux/x86_64               | ✅        |
| Build airbyte/normalization:dev                                                 | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-bigquery:integrationTest | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/edgao_fix_bq/1688425603/4d24db8061f84da73d1a1987853969b97680a3ba/destination-bigquery/1.5.2/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-bigquery-denormalized test report (commit [`4d24db8061`](https://github.com/airbytehq/airbyte/pull/27856/commits/4d24db8061f84da73d1a1987853969b97680a3ba)) - ❌

⏲️  Total pipeline duration: 18mn22s 

| Step                                                                                         | Result   |
|:---------------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-bigquery-denormalized/metadata.yaml     | ✅        |
| Connector version semver check                                                               | ✅        |
| QA checks                                                                                    | ✅        |
| Build connector tar                                                                          | ✅        |
| Build destination-bigquery-denormalized docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-bigquery-denormalized:integrationTest | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/edgao_fix_bq/1688428519/4d24db8061f84da73d1a1987853969b97680a3ba/destination-bigquery-denormalized/1.5.2/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery-denormalized test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-bigquery test report (commit [`4d24db8061`](https://github.com/airbytehq/airbyte/pull/27856/commits/4d24db8061f84da73d1a1987853969b97680a3ba)) - ✅

⏲️  Total pipeline duration: 04mn18s 

| Step                                                                            | Result   |
|:--------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-bigquery/metadata.yaml     | ✅        |
| Connector version semver check                                                  | ✅        |
| Connector version increment check                                               | ✅        |
| QA checks                                                                       | ✅        |
| Build connector tar                                                             | ✅        |
| Build destination-bigquery docker image for platform linux/x86_64               | ✅        |
| Build airbyte/normalization:dev                                                 | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-bigquery:integrationTest | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/edgao_fix_bq/1688428519/4d24db8061f84da73d1a1987853969b97680a3ba/destination-bigquery/1.5.2/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-bigquery test report (commit [`f04731b8c2`](https://github.com/airbytehq/airbyte/pull/27856/commits/f04731b8c2053377ff4cf16266e50852d4016df2)) - ✅

⏲️  Total pipeline duration: 27mn09s 

| Step                                                                            | Result   |
|:--------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-bigquery/metadata.yaml     | ✅        |
| Connector version semver check                                                  | ✅        |
| Connector version increment check                                               | ✅        |
| QA checks                                                                       | ✅        |
| Build connector tar                                                             | ✅        |
| Build destination-bigquery docker image for platform linux/x86_64               | ✅        |
| Build airbyte/normalization:dev                                                 | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-bigquery:integrationTest | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/edgao_fix_bq/1688589099/f04731b8c2053377ff4cf16266e50852d4016df2/destination-bigquery/1.5.2/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-bigquery-denormalized test report (commit [`f04731b8c2`](https://github.com/airbytehq/airbyte/pull/27856/commits/f04731b8c2053377ff4cf16266e50852d4016df2)) - ✅

⏲️  Total pipeline duration: 17mn06s 

| Step                                                                                         | Result   |
|:---------------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-bigquery-denormalized/metadata.yaml     | ✅        |
| Connector version semver check                                                               | ✅        |
| QA checks                                                                                    | ✅        |
| Build connector tar                                                                          | ✅        |
| Build destination-bigquery-denormalized docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-bigquery-denormalized:integrationTest | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/edgao_fix_bq/1688589099/f04731b8c2053377ff4cf16266e50852d4016df2/destination-bigquery-denormalized/1.5.2/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery-denormalized test
```

**Pull request merged**